src/Puppeteer.ts

```diff
@@ -122,26 +187,86 @@ export class Puppeteer {
     return this._lazyLauncher;
   }
 
+  /**
+   * @returns The name of the browser that is under automation (`"chrome"` or `"firefox"`)
+   *
+   * @remarks
+   * The product is set by the `PUPPETEER_PRODUCT` environment variable or the `product` option in `puppeteer.launch([options])` and defaults to `chrome`. Firefox support is experimental.
+   */
   get product(): string {
     return this._launcher.product;
   }
 
+  /**
+   * @remarks
+   * @example
+   *
+   * ```js
+   * const puppeteer = require('puppeteer');
+   * const iPhone = puppeteer.devices['iPhone 6'];
+   *
+   * (async () => {
+   *   const browser = await puppeteer.launch();
+   *   const page = await browser.newPage();
+   *   await page.emulate(iPhone);
+   *   await page.goto('https://www.google.com');
+   *   // other actions...
+   *   await browser.close();
+   * })();
+   * ```
+   * **NOTE** The old way (Puppeteer versions <= v1.14.0) devices can be obtained with `require('puppeteer/DeviceDescriptors')`.
```

Comment:
We actually shouldn't say this (I know this was in the old docs and I'll remove it from there) because we shouldn't support importing anything other than `puppeteer` (e.g. our folder structure is subject to change) - so could we lose this line ?